exasol/toolbox/nox/_dependencies.py

```diff
@@ -263,7 +263,7 @@ def _parse_args(session) -> argparse.Namespace:
     def run(self, session: Session) -> None:
         args = self._parse_args(session)
 
-        command = ["poetry", "run", "pip-audit", "-f", "json"]
+        command = ["pip-audit", "-f", "json"]
```

Comment:
to reviewer: many of these commands are not tested in the CI, so please consider verifying locally that you get the same expected behavior with the removal of "poetry", "run".